sentry-kotlin-extensions/src/main/java/io/sentry/kotlin/SentryContext.kt

```diff
@@ -9,26 +9,32 @@ import kotlin.coroutines.CoroutineContext
 /**
  * Sentry context element for [CoroutineContext].
  */
-public class SentryContext(private val hub: IHub = Sentry.getCurrentHub().clone()) :
-    CopyableThreadContextElement<IHub>, AbstractCoroutineContextElement(Key) {
+@SuppressWarnings("deprecation")
+// TODO fork instead
```

Comment:
Should be replaced by forking scopes in a follow up PR. There's more TODOs in this PR that should be cleaned up in follow up PRs.